### PR TITLE
fix: Fixed display of PeerInfoTitle in AdminModal

### PR DESCRIPTION
**Pull request by pvlv**

### CODE REVIEW

**Comment by MatveyNeyman** [comment]
LGTM

File: src/components/AdminModal/AdminModal.css [hunk 1]
```diff
@@ -35,6 +35,11 @@
   margin-right: 16px;
 }
 
+.info {
+  width: 0;
```
Comment (olegshilov) [inline src/components/AdminModal/AdminModal.css]: use `flex-fix` mixin for this.
https://github.com/dialogs/dialog-web-components/blob/master/src/styles/helpers.css#L24

Comment (pvlv) [reply src/components/AdminModal/AdminModal.css]: fixed

**Comment by olegshilov** [comment]
revert `package-lock.json`

**Pull request merged**